README.md

```diff
@@ -46,7 +66,176 @@ Pipelines:
 export JF_PRODUCT_DATA_INTERNAL=/opt/jfrog/pipelines/var/
 ````
 
-## Fluentd Install
+## Fluentd Configuration for Datadog
+
+Download and configure the relevant fluentd.conf files for Datadog
+
+### Configuration steps for Artifactory
+
+Download the artifactory fluentd configuration file to a directory the user has permissions to write, such as the $JF_PRODUCT_DATA_INTERNAL locations discussed above in the [Environment Configuration](#environment-configuration) section.
+
+````text
```

Comment:
Include the export line for this product right above the cd so that all the steps for artifactory are in the same place.

Reply:
This way, we'll have the export step for every platform for both configuration and installation which is redundant (5 configurations, 4 installations, original 5 steps)

Reply:
We should say here that 
Please ensure that you have exported the $JF_PRODUCT_DATA_INTERNAL environment variable to point to the path for the product you need to the logs from. viz.

export JF_PRODUCT_DATA_INTERNAL=<jfrog product specific path> # Refer to the Environment section to find the path

Does that make sense?